src/components/common/auth/leftDiv.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const LeftDiv = ({ Text }) => (
```

Comment:
Why does `Text` begin with a capital letter?

Reply:
okay I would correct it